src/integrate/concat/script.py

```diff
@@ -189,7 +186,7 @@ def set_conflicts(concatenated_data: mu.MuData,
             getattr(mod, conflict_matrix_name)[conflict_name] = conflict_data.sort_index()
     return concatenated_data
 
-def concatenate_modalities(modalities: dict[str, anndata.AnnData],
+def concatenate_modalities(sample_ids: tuple[str], modalities: dict[str, list[anndata.AnnData]],
```

Comment:
I think I made a mistake here. A general `Iterable[anndata.Anndata]` is better here